specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2025-06-01-preview/sqlServerDatabases.json

```diff
@@ -0,0 +1,777 @@
+      "readOnly": true,
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "appliesToMigrationTargetPlatform": {
+            "type": "string"
+          },
+          "featureId": {
+            "type": "string"
+          },
+          "issueCategory": {
+            "type": "string"
+          },
+          "moreInformation": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "TargetReadiness": {
+      "description": "The target readiness for migration for this database.",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "azureSqlDatabase": {
+          "$ref": "#/definitions/SkuRecommendationSummary"
+        },
+        "azureSqlManagedInstance": {
+          "$ref": "#/definitions/SkuRecommendationSummary"
+        },
+        "azureSqlVirtualMachine": {
+          "$ref": "#/definitions/SkuRecommendationSummary"
+        }
+      }
+    },
+    "SkuRecommendationSummary": {
+      "description": "The SKU recommendation summary.",
+      "type": "object",
+      "properties": {
+        "numOfBlockerIssues": {
+          "$ref": "#/definitions/SkuRecommendationResultsNumberOfBlockerIssues"
+        },
+        "recommendationStatus": {
+          "$ref": "./common.json#/definitions/SkuRecommendationResultsRecommendationStatus"
+        },
+        "impactedObjectsSummary": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "./common.json#/definitions/ImpactedObjectsInfo"
+          },
+          "x-ms-identifiers": []
+        },
+        "monthlyCost": {
+          "$ref": "./common.json#/definitions/SkuRecommendationResultsMonthlyCost"
+        },
+        "monthlyCostOptions": {
+          "$ref": "./common.json#/definitions/SkuRecommendationResultsMonthlyCostOptions"
+        },
+        "targetSku": {
+          "type": "object",
+          "properties": {
+            "category": {
+              "type": "object",
+              "properties": {
+                "computeTier": {
+                  "type": "string",
+                  "description": "The compute tier of the target SKU."
+                },
+                "hardwareType": {
+                  "type": "string",
+                  "description": "The hardware type of the target SKU."
+                },
+                "sqlPurchasingModel": {
+                  "type": "string",
+                  "description": "The SQL purchasing model of the target SKU."
+                },
+                "sqlServiceTier": {
+                  "type": "string",
+                  "description": "The SQL service tier of the target SKU."
+                },
+                "zoneRedundancyAvailable": {
+                  "type": "boolean",
+                  "description": "Indicates if zone redundancy is available for the target SKU."
+                }
+              }
+            },
+            "computeSize": {
```

Comment:
>computeSize [](http://example.com/codeflow?start=13&length=11)

computeVCores? 

Let the name and description give a better idea of what the user should expect.